src/main/java/uk/gov/hmcts/reform/civil/service/SendFinalOrderPrintService.java

```diff
@@ -56,4 +84,205 @@ public void sendJudgeFinalOrderToPrintForLIP(String authorisation, Document post
                                      SendFinalOrderPrintService.FINAL_ORDER_PACK_LETTER_TYPE, recipients);
 
     }
+
+    public void sendJudgeTranslatedOrderToPrintForLIP(String authorisation, Document orderDocument, CaseData caseData, CaseEvent caseEvent) {
+
+        List<String> recipients = new ArrayList<>();
+        CaseData civilCaseData = caseDetailsConverter
+            .toCaseData(coreCaseDataService
+                            .getCase(Long.parseLong(caseData.getGeneralAppParentCaseLink().getCaseReference())));
+
+        DocmosisDocument coverLetter = generate(caseData, civilCaseData, caseEvent);
+
+        CaseDocument coverLetterCaseDocument = documentManagementService.uploadDocument(
+            authorisation,
+            new PDF(
+                POST_ORDER_COVER_LETTER_LIP.getDocumentTitle(),
+                coverLetter.getBytes(),
+                DocumentType.POST_ORDER_COVER_LETTER_LIP
+            )
+        );
+
+        List<DocumentMetaData> documentMetaDataList
+            = stitchCoverLetterAndOrderDocument(coverLetterCaseDocument, orderDocument);
+
+        CaseDocument stitchedDocument = civilDocumentStitchingService.bundle(
```

Comment:
Here, you need to check if (stitch_is_enabled) and then only call the stitching API; otherwise, this will fail on a lower environment.

Reply:
now checking for stitching enabled